tutorials/introductory/lifecycle.py

```diff
@@ -41,7 +41,8 @@
 .. note::
 
    In general, try to use the object-oriented interface over the pyplot
```

Comment:
```suggestion
   In general, the explicit interface is preferred over the pyplot
```

Reply:
That would require rephrasing the earlier paragraphs in the tutorial as well. That may be a good thing to do but wasn't my intention here. Is this enough of an incremental improvement to stand on its own?

Reply:
`explicit object-oriented..`?

Reply:
That's better but it is still a break from how the opening paragraph describes things. But perhaps Im worrying too much about that

Reply:
Maybe fix the first parapraph while we are here?  It is not terribly clear.